libraries/lib-shuttlegui/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+#[[
+An embedded language for describing dialog layouts, and also alternative
+visitations of the layout description to construct a dialog, or to shuttle
+values between its controls and program variables or preferences.
+
+Some good properties that this library does not have, but that the hoped for
```

Comment:
I'm not sure, that commenting a possible future makes sense here. 

To be fair, after working much more time on Audacity since I proposed that we can do some replacement I don't think it would be feasible. We won't be able to keep any of UI related code either way.

Reply:
I have not yet given up that hope.